mcr_scene_segmentation/ros/config/scene_segmentation_constraints.yaml

```diff
@@ -1,12 +1,12 @@
 # reasonable parameters to find planes of @Work workspaces
 scene_segmentation:
     voxel_leaf_size: 0.02
-    voxel_filter_field_name: z
-    voxel_filter_limit_min: -0.15
-    voxel_filter_limit_max: 0.2
-    passthrough_filter_field_name: x
-    passthrough_filter_limit_min: 0.0
-    passthrough_filter_limit_max: 0.8
+    passthrough_limit_min_x: 0.0
```

Comment:
As I mentioned in the prev PR, rename these params to avoid confusion (i.e. voxel_filter_limit...)

Reply:
yeah that's reasonable, will do that later today